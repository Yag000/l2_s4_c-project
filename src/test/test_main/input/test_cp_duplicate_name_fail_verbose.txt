mkdir t
mkdir k
cp k t
