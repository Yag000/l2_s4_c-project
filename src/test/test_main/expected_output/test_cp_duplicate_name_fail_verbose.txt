Parsing file src/test/test_main/input/test_cp_duplicate_name_fail_verbose.txt ...
/$ mkdir t
/$ mkdir k
/$ cp k t
Invalid name : an element with the same name already exists.
